images/k8s-cloud-builder/Dockerfile

```diff
@@ -44,32 +41,60 @@ RUN apt-get -q update \
 # Install gcloud
 # common::set_cloud_binaries() looks for it in this path
 RUN mkdir /opt/google
-RUN curl -sSL https://sdk.cloud.google.com > /tmp/install.sh && \
-    bash /tmp/install.sh --install-dir=/opt/google --disable-prompts
+RUN apt-get -y update \
+    && apt-get -y install \
+        gcc \
+        python2.7 \
+        python-dev \
+        python-setuptools \
+        wget \
+        ca-certificates \
+        # These are necessary for add-apt-respository
+        software-properties-common \
+    # Install Git >2.0.1
+    && apt-get -y update \
+    && apt-get -y install git \
+    # Setup Google Cloud SDK (latest)
+    && curl -sSL https://sdk.cloud.google.com > /tmp/install.sh \
+    && bash /tmp/install.sh --install-dir=/opt/google --disable-prompts \
```

Comment:
A nit, but could this be gotten via apt-get also?

Reply:
@tpepper -- yep, it can. This Dockerfile is Frankenstein work from a few strewn across other repos (that are known to work). I'm going to squash the layers (as you can see some repetition of commands as well) in a follow-up PR.